src/Microsoft.Framework.Logging.Interfaces/ILoggerFactory.cs

```diff
@@ -16,5 +16,7 @@ public interface ILoggerFactory
         ILogger Create(string name);
 
         void AddProvider(ILoggerProvider provider);
+
+        LogLevel MinimumLevel { get; set; }
```

Comment:
Code style: Generally, properties go after ctors and before methods. Should update the other files too.